### PR TITLE
chore: enable e2e pipeline run for LL

### PR DESCRIPTION
**Pull request by jhockett**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.base.yml [hunk 1]
```diff
@@ -569,6 +569,7 @@ workflows:
             branches:
               only:
                 - master
+                - layers-rework
```
Comment (yuth) [inline .circleci/config.base.yml:572]: Do we need to run these tests on layers

File: .circleci/config.base.yml [hunk 2]
```diff
@@ -591,6 +592,7 @@ workflows:
...
```
Comment (yuth) [inline .circleci/config.base.yml:595]: Same here

**Review by yuth — APPROVED** [review]
minor nits

File: .circleci/config.base.yml (hunk 1 above)
Comment (jhockett) [reply .circleci/config.base.yml:572]: Whoops, this is a mistake. Meant to exclude these from the nightly tests, good catch!

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "publish:beta": "lerna publish --exact --dist-tag=beta --preid=beta --conventional-commits --conventional-prerelease --message 'chore(release): Publish [ci skip]' --no-verify-access --yes",
     "publish:release": "lerna publish --conventional-commits --exact --yes --message 'chore(release): Publish [ci skip]' --no-verify-access",
     "postpublish:release": "git fetch . release:master && git push origin master",
+    "publish:layers": "lerna publish --exact --dist-tag=layers --preid=alpha --conventional-commits --conventional-prerelease --message 'chore(release): Publish [ci skip]' --no-verify-access --yes",
```
Comment (SwaySway) [inline package.json:37]: I think this needs to be renamed by the branch name since the deploy process will run `npm run publish:$CIRCLE_BRANCH`.

Comment (yuth) [reply package.json:37]: Yes, it has to match the branch name

**Pull request closed without merging**